in_toto/util.py

```diff
@@ -1,62 +1,53 @@
 import sys
 import getpass
+from enum import Enum
 
 import in_toto.gpg.functions
 import in_toto.gpg.formats
 import securesystemslib.formats
 import securesystemslib.hash
+import securesystemslib.interface
 import securesystemslib.keys
 import securesystemslib.exceptions
 
+from securesystemslib.interface import (import_ed25519_privatekey_from_file,
+                                        import_ed25519_publickey_from_file)
+
 DEFAULT_RSA_KEY_BITS = 3072
 
-def generate_and_write_rsa_keypair(filepath, bits=DEFAULT_RSA_KEY_BITS,
```

Comment:
This was a completely redundant copy of the same function in `securesystemslib.interface`